_includes/content/ssh-gen-keys.md

```diff
@@ -0,0 +1,106 @@
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#mac-key-gen" data-group="mac">Mac</a></li>
+  <li><a data-toggle="tab" data-target="#win-key-gen" data-group="win">Windows</a></li>
+  <li><a data-toggle="tab" data-target="#linux-key-gen" data-group="linux">Linux</a></li>
+</ul>
+<div class="tab-content">
+<div id="mac-key-gen" class="tab-pane fade in active">
+<br>
+{% capture mac-content-gen %}
+1.  Open a command-line terminal.
+
+2.  Paste the text below, substituting in your GitHub email address.
+
+    ```none
+    ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
```

Comment:
You might want to show the option to control the file name of the keys, so they are not just the default.  The flag is `-f`. So you might do like `-f docker-key`

It's a good idea to have a key per area of concern, rather than one key for your whole life. This way if you need to rotate or revoke a key later, it is minimally disruptive.